js/main.js

```diff
@@ -77,25 +105,50 @@ function toggleNoEntries() {
   }
 }
 
-const $anchorTag = document.querySelector('.anchor-tag');
 $anchorTag.addEventListener('click', function () {
   viewSwap('entries');
 }
 );
 
-const $anchorTagTwo = document.querySelector('.new-button');
 $anchorTagTwo.addEventListener('click', function () {
   viewSwap('entry-form');
+  const $notes = document.querySelector('#notes');
+  $notes.innerHTML = '';
```

Comment:
This shouldn't be needed. If you are still running into issues with the `textarea` being filled in, the issue should be coming from some place else